static_src/components/header.jsx

```diff
@@ -28,9 +28,9 @@ export default class Header extends React.Component {
       <Action href="/handshake" label="Login" type="outline">
         Login
       </Action>
-    </HeaderLink>:
+    </HeaderLink> :
```

Comment:
is there a reason for the extra space beyond the tag? (style question i don't know)